parlai/agents/transformer/transformer.py

```diff
@@ -26,13 +26,17 @@ def add_common_cmdline_args(argparser):
     argparser.add_argument('-esz', '--embedding-size', type=int, default=300,
                            help='Size of all embedding layers')
     argparser.add_argument('-nl', '--n-layers', type=int, default=2)
-    argparser.add_argument('-hid', '--ffn-size', type=int, default=300,
+    argparser.add_argument('-hs', '--ffn-size', type=int, default=300,
```

Comment:
why did we change this?

Reply:
Corresponding to args for seq2seq. I could add both. The full option is the same.